oslo_cache/backends/redis.py

```diff
@@ -0,0 +1,45 @@
+from dogpile.cache.backends import redis as redis_backend
+from oslo_cache import core
+__all__ = [
+    'RedisSentinelTLSBackend'
+]
+class RedisSentinelTLSBackend(redis_backend.RedisSentinelBackend):
+    def __init__(self,arguments):
+        """
+        Wrapper class for dogpile.cache.redis_sentinel.
+        Currently only translates arguments for dogpile.cache.backends.RedisSentinelBackend
+        because user is unable to pass them using oslo config in it's current format
+        arguments:
+           sentinels: list of sentinels formated as sentinel1IP:port,sentinel2IP:port
+           ssl: turns on TLS
+           ssl-<keyfile,certfile,ca_certs>: TLS certificate, key and CA
+           username,password: login credentials 
+        """
```

Comment:
You should probably rewrite the docstring to be in Google docstring format so it is consistent with other backends already in oslo:

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

https://github.com/SovereignCloudStack/oslo.cache/blob/17b2fdb89fa166adccb1be1dc42a4682cb6118ad/oslo_cache/backends/mongo.py#L40-L155